test.sh

```diff
@@ -0,0 +1,251 @@
+#!/bin/bash
+set -e -o pipefail
+
+help() {
+    echo 'Usage ./test.sh [-f docker-compose.yml] [-p project] [args]'
+    echo
+    echo 'Optional args'
+    echo '  run:            [default] starts up the entire stack and runs the test clients.'
+    echo '  check:          verify your local environment is correctly configured.'
+    echo '  show:           open web pages of an already running stack.'
+    echo '  test <logtype>: run test client against an already running stack. logtype'
+    echo '                  should be one of: syslog, gelf'
+    echo '  help            help. you are reading it now.'
+    echo
+    echo 'Optional flags:'
+    echo '  -f <filename>   use this file as the docker-compose config file'
+    echo '  -p <project>    use this name as the project prefix for docker-compose'
+    echo
+}
+
+
+# default values which can be overriden by -f or -p flags
+export COMPOSE_PROJECT_NAME=elk
+export COMPOSE_FILE=
+
+# give the docker remote api more time before timeout
+export COMPOSE_HTTP_TIMEOUT=300
+
+# populated by `check` function whenever we're using Triton
+TRITON_USER=
+TRITON_DC=
+TRITON_ACCOUNT=
+
+# ---------------------------------------------------
+# Top-level commmands
+
+run() {
+    check
+    docker-compose up -d \
```

Comment:
Yah, that was left over from the original version where we had the Nginx stuffed into the same `docker-compose.yml` file. Will fix.